src/compose.ts

```diff
@@ -294,6 +294,12 @@ function normalizeService(
 				service.tmpfs = service.tmpfs.concat(tmpfs);
 			}
 		}
+		if (labels.length > 0) {
+			service.labels = {
+				...labels.reduce((o, l) => Object.assign(o, { [l]: 1 }), {}),
```

Comment:
This might be cleaner?

```suggestion
				..labels.reduce((o, l) =>  ({...o, [l]: 1 }), {}),
```